packages/dashboard-frontend/src/store/__mocks__/devWorkspaceBuilder.ts

```diff
@@ -10,33 +10,41 @@
  *   Red Hat, Inc. - initial API and implementation
  */
 
-import { IDevWorkspace } from '../../services/workspace-client/devworkspace/types';
+import devfileApi from '../../services/devfileApi';
 import getRandomString from '../../services/helpers/random';
 import { DevWorkspaceStatus, WorkspaceStatus } from '../../services/helpers/types';
 
 export class DevWorkspaceBuilder {
 
-  private workspace: IDevWorkspace = {
+  private workspace: devfileApi.DevWorkspace = {
     kind: 'DevWorkspace',
     apiVersion: 'workspace.devfile.io/v1alpha2',
     metadata: {
+      annotations: {},
+      labels: {},
       name: 'dev-wksp-' + getRandomString(4),
       namespace: '',
+      uid: 'uid-' + getRandomString(4)
     },
     spec: {
       started: false,
       routingClass: 'che',
       template: {},
     },
-    status: {
```

Comment:
I am sure we should leave 'status' as an empty object

```
status: { }
```

and remove things like that:
```
if (this.workspace.status === undefined) {
      this.workspace.status = this.buildStatus();
    }
```